plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -37,11 +35,9 @@ protected QueryRunner createQueryRunner()
     {
         return HiveQueryRunner.builder()
                 .setHiveProperties(ImmutableMap.of(
-                        "hive.allow-register-partition-procedure", "true",
                         // Reduce writer sort buffer size to ensure SortingFileWriter gets used
                         "hive.writer-sort-buffer-size", "1MB",
```

Comment:
I think this config is also unnecessary. In some other test it was meant ensure that file sorting doesn't fit in memory and SortingFileWriter gets used. That shouldn't have relevance here, unless I missed something
cc: @JamesRTaylor

Reply:
i don't have opinion. i think it's ok for follow up.